packages/core/src/components/textarea/test/read-only/textarea.e2e.ts

```diff
@@ -0,0 +1,38 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/textarea/test/read-only/index.html';
+
+test.describe('tds-textarea', () => {
+  test('renders read-only textarea correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const tdsTextarea = page.locator('tds-textarea');
+    expect(tdsTextarea).toBeTruthy();
+
+    /* Expect the tds-textarea to have the read-only attribute */
+    await expect(tdsTextarea).toHaveAttribute('read-only');
+    /* Expect no diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('read-only textarea - native textarea should have readonly attribute', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const textarea = page.locator('tds-textarea').locator('textarea');
+    /* Expect the textarea within tds-textarea to have the readonly attribute */
+    await expect(textarea).toHaveAttribute('readonly');
+    const readonlyIcon = page.locator('tds-icon[name="edit_inactive"]');
+    /* Expect a tds-icon to be present when readonly mode is active */
+    expect(readonlyIcon).not.toBeNull();
+  });
+
+  // Error: locator.fill: Target closed
+  // Not able to update the since we get Playwright error here when trying to input a readonly input.
+  // Skipping for now.
+  test.skip('Type text in textarea - no possibility to type text', async ({ page }) => {
```

Comment:
Could we test that the component does not receive a click event instead to test that it's not interactable?

Reply:
Hmm I tried to do this, Its just that the `await textarea.click()` does get the error. I wonder if just checking if it has the disabled attribute for now.

Reply:
That could be a way to do it. Maybe this should be discussed with Tatyana though?

Reply:
Maybe we could test for that when you try and type text into it, it remains empty?

```
await textarea.type('Adding some text');
  
 // Check that the textarea remains empty (no text was entered)
 expect(await textarea.inputValue()).toBe('');`
```

Reply:
If we wanna test also for the disabled attribute we could do:

```
const isDisabled = await textarea.getAttribute('disabled');
expect(isDisabled).toBeTruthy();
```

Reply:
> Maybe we could test for that when you try and type text into it, it remains empty?
> 
> ```
> await textarea.type('Adding some text');
>   
>  // Check that the textarea remains empty (no text was entered)
>  expect(await textarea.inputValue()).toBe('');`
> ```

I still get `Error: locator.fill: Target closed` when trying to do stuff to a readonly field. might just check for the readonly tag. Since its not disabled, its "readonly"